addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+    <config-description uri="thing-type:netatmo:bridge">                         
+        <parameter name="clientId" type="text">
+            <label>Client ID</label>
+            <description>Client ID provided for the application you created on http://dev.netatmo.com/createapp</description>
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="clientSecret" type="text">
+            <label>Client Secret</label>
+            <description>Client Secret provided for the application you created</description>
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="username" type="text">
+            <label>Username</label>
+            <description>Your Netatmo API username (email)</description>
+            <required>true</required>
+            <advanced>false</advanced>
+        </parameter>
+            
+        <parameter name="password" type="text">
+            <label>Password</label>
```

Comment:
@maggu2810 This might be interesting for you regarding our discussion on https://github.com/eclipse/smarthome/pull/954 - the different GSON versions seem to be pretty different after all, so we will need to find a clear strategy.